src/commands/folder/spec/folder.spec.server.ts

```diff
@@ -0,0 +1,191 @@
+import dotenv from 'dotenv'
+import { folder } from '../index'
+import { generateTimestamp } from '../../../utils/utils'
+import { ServerType, Target } from '@sasjs/utils/types'
+import {
+  removeFromGlobalConfig,
+  saveToGlobalConfig
+} from '../../../utils/config'
+import { Command } from '../../../utils/command'
+
+describe('sasjs job execute', () => {
+  let target: Target
+
+  beforeAll(async (done) => {
+    target = await createGlobalTarget()
+    process.projectDir = process.cwd()
+
+    done()
+  })
+
+  afterAll(async (done) => {
+    await removeFromGlobalConfig(target.name)
+    done()
+  })
+
+  it('list folder children', async (done) => {
+    const timestamp = generateTimestamp()
+    const testFolderPath = `/Public/app/cli-tests/cli-tests-folder-${timestamp}`
```

Comment:
`testFolderPath` is the same for every test suit, but with a different `timestamp`, so you can create a function returning `testFolderPath`